website/blog/2021-03-16-64updates.md

```diff
@@ -0,0 +1,203 @@
+Fret not! We’re continually reassessing and reviewing the set of modules that are important to React Native for Windows devs, so if you need a module that doesn’t yet have a Windows implementation, please [open a new issue on the React Native for Windows GitHub repo.](https://github.com/microsoft/react-native-windows/issues/new/choose)
+
+If you have experience writing Windows native code, you can also use the [new Community module template workflow](https://microsoft.github.io/react-native-windows/docs/next/native-modules-setup) to create a module template for Windows. This is a great way to start using native Windows APIs quickly in custom modules.
+
+## 🤝 Janea Systems
+![janea-logo](assets/janeasystemslogo.png)
+
+Finally, we wanted to highlight our partnership with [Janea Systems](http://www.janeasystems.com/), who has helped provide implementation, consultation, and support for many of the modules listed above. In particular, we wanted to shout out to [@jaimecbernardo](https://github.com/jaimecbernardo), [@joaocgreis](https://github.com/joaocgreis), and [@bzoz](https://github.com/bzoz) from Janea Systems for their help over the last few months.
+
+# Breaking changes
+
+- **Native module thread affinity**
+
+ A bug was discovered which led to custom native modules often being invoked on the UI thread. This was unintentional and has been changed. We recognize that this may break existing modules in cases where affinity was relied upon without explicitly queuing to the UI thread. If you're affected by this or have a module that is, you can use the [`UIDispatcher` API](https://microsoft.github.io/react-native-windows/docs/ireactcontext-api#uidispatcher) to post to the UI thread.
+
+- `acceptsKeyboardFocus` **removed**
+
+ In 0.63 we started warning on usage of `acceptsKeyboardFocus` that the property would be replaced by the built-in and type-safe focusable. In 0.64 we removed support from the property, redboxing if it is passed to a view and ignoring the property on touchables.
+
+- `AppTheme` **API reconciliation with Appearance**
+
+ React Native 0.62 introduced Appearance and `useColorScheme` hooks to respond to changes in light/dark mode. This functionality supersedes several APIs in the react-native-windows specific `AppTheme` library. `AppTheme` functions that warned about removal in 0.63 are removed in 0.64. [See more here](https://microsoft.github.io/react-native-windows/docs/apptheme-api) on how to write components that adapt to theme.
+
+- `EmitJSEvent` **parameter passing**
+
+ Calls to EmitJSEvent on the ReactContext previously wrapped the event object in a JavaScript array instead of passing it 1:1. This was fixed in 0.64, but any existing consumption of events fired by EmitJSEvent must be updated.
+
+- `ref.focus()` **focus visuals**
+
+ Calling `focus()` on a component ref will now show native focus visuals by default. If you don't want focus visuals around your component, you can set the property `enableFocusRing={false}`.
+

```

Comment:
Hi! Not a comment on the PR but where would I be able to find more info/implementation details of this API? I'm working on some key view loop / focus stuff on rn-macos and wanted to see how this worked. Thanks =)

Reply:
@saadnajmi please file an issue on the repo